scripts/scr_weapon/scr_weapon.gml

```diff
@@ -2011,6 +2062,53 @@ global.gear = {
 				"artifact": 16 // Augmented
 			},
 		},
+		"Light Power Armour":{
+			"abbreviation": "LPArm",
+			"description": "A type of power armour that can be used by regular humans.",
+			"armour_value": { // TODO - needs rebalancing, also in scr_weapon_en and other places
+				"standard": 20,
+				"master_crafted": 22,
+				"artifact": 25
+			},
+		},
+		"Ranger Armour":{
+			"abbreviation": "RngrArm",
+			"description": "This armour is used by eldar rangers.",
+			"armour_value": {
+				"standard": 25,
+				"master_crafted": 27,
+				"artifact": 30
+			},
+		},
```

Comment:
You may want to transfer this into that pr then.

Reply:
Alright.